barretenberg/cpp/src/barretenberg/plonk_honk_shared/arithmetization/mega_arithmetization.hpp

```diff
@@ -83,14 +83,14 @@ template <typename FF_> class MegaArith {
         {
             this->ecc_op = 1 << 10;
             this->pub_inputs = 1 << 7;
-            this->arithmetic = 187000;
+            this->arithmetic = 201000;
```

Comment:
had to update this with the change, and also reduce some other gates to prevent increasing the circuit size to 2^20.